runtime/permissions/prompter.rs

```diff
@@ -185,6 +187,16 @@ impl PermissionPrompter for TtyPrompter {
       eprint!("\x1B[{}A\x1B[0J", n);
     }
 
+    fn is_unary_permission(name: &str) -> bool {
```

Comment:
Feel like this should be a boolean passed to `permission_prompt` instead of doing this check here

Reply:
Do you mean where `is_unary_permission()` is defined or used? I thought where I had it defined and used made sense, as it's used to adjust the prompt message (see line 209) and handle invalid prompt responses (see lines 253 and 262).

Reply:
I mean that `permissions_prompt` should have following signature:
```
pub fn permission_prompt(
  message: &str,
  flag: &str,
  api_name: Option<&str>,
  is_unary: bool
) -> PromptResponse {
```
Matching against string names seems a bit brittle to me. But maybe I'm overreacting here.

Reply:
Oh, I see. Yeah, that's definitely a better way to do it.